### PR TITLE
Add new error schema to extendDuration: SessionStateInConflict409 with QUALITY_ON_DEMAND.SESSION_EXTENSION_NOT_ALLOWED 

### LINKED ISSUE #351: Behaviour when extending not AVAILABLE session
**Problem description**
Current spec do not indicate the expected behaviour when the extension of the duration of a session in status other than AVAILABLE is requested

**Expected action**
Agree on the expected result, and update spec and tests. 

For sessions with qosStatus = REQUESTED, implementation may accept the request and take into account the petition when the session is activated, or return a success without modifying the original duration (ignoring the request), or even return an error indicating that this request is not acceptable  in  the current state. 
For sessions with qosStatus = UNAVAILABLE, implementation may return a success without modifying the duration (ignoring the request), or return an error indicating that this request is not acceptable  in the current state.

### PR DESCRIPTION
**Pull request by hdamker**
#### What type of PR is this?

* correction

#### What this PR does / why we need it:

Clarifies the undefined behaviour if API consumer tries to extend a session which is state REQUESTED or UNAVAILABLE as discussed in #351 

#### Which issue(s) this PR fixes:

Fixes #351

### CODE REVIEW

**Review by jlurien — APPROVED** [review]
LGTM

File: code/API_definitions/quality-on-demand.yaml [hunk 1]
```diff
@@ -1175,6 +1177,23 @@ components:
...
+    SessionStatusConflict409:
+      description: Conflict
+      headers:
+        x-correlator:
+          $ref: '#/components/headers/x-correlator'
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/ErrorInfo"
+          example:
```
Comment (jlurien) [inline code/API_definitions/quality-on-demand.yaml:1189]:
```suggestion
          examples:
```

**Review by jlurien — APPROVED** [review]
LGTM

**Comment by hdamker** [comment]
@eric-murray @RandyLevensalor please have as well a look on this last minute change - should be ok, but more eyes might see more.

**Review by RandyLevensalor — APPROVED** [review]

**Pull request merged**